census/census.go

```diff
@@ -127,15 +212,24 @@ func (m *Manager) Init(storageDir, rootAuthPubKey string) error {
 	return nil
 }
 
+// Deinit terminates the Manager goroutines and closes the database.  The
+// Manager can't be used after Deinit is called.
+func (m *Manager) Deinit() error {
```

Comment:
Usually this is called Shutdown :)

Reply:
Speaking of which, Init should be called Start if it fires goroutines. Because Init has a connotation that it just initializes memory or storage, not that it starts background processes.

Reply:
Renamed Init to Start, and Deinit to Stop.